ci/scripts/python_wheel_windows_test.bat

```diff
@@ -58,11 +58,14 @@ py -0p
 @REM Validate wheel contents
 %PYTHON_CMD% C:\arrow\ci\scripts\python_wheel_validate_contents.py --path C:\arrow\python\repaired_wheels || exit /B 1
 
-@rem Download IANA Timezone Database for ORC C++
-curl https://cygwin.osuosl.org/noarch/release/tzdata/tzdata-2024a-1.tar.xz --output tzdata.tar.xz || exit /B
-mkdir %USERPROFILE%\Downloads\test\tzdata
-arc unarchive tzdata.tar.xz %USERPROFILE%\Downloads\test\tzdata || exit /B
-set TZDIR=%USERPROFILE%\Downloads\test\tzdata\usr\share\zoneinfo
+@rem Install tzdata python package and set TZDIR to tzdata database defined inside
+%PYTHON_CMD% -m pip install tzdata || exit /B 1
```

Comment:
Can we do this in `ci/docker/python-wheel-windows-test-vs2019.dockerfile`?

Ah, wait. It seems that we already did it: https://github.com/apache/arrow/blob/2cbc7d696cef76b61391064b0395b15edcc246a3/python/requirements-wheel-test.txt#L7

Reply:
Good catch, I'll remove the pip install from the script.